bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -240,16 +243,10 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
     }
 
     private String applyUrlRewrites(String url) {
-        for (String rewrite : urlRewrites) {
-            if (!rewrite.contains(":") || rewrite.startsWith(":")) {
-                continue;
-            }
-            String[] tokens = rewrite.split(":");
-            String path = tokens[0];
-            String replacement = tokens.length > 1 ? tokens[1] : "";
-            if (url.contains(path)) {
-                url = url.replaceFirst(path, replacement);
-                break;
+        String path = URI.create(url).getPath();
```

Comment:
@leonbe02 LGTM - just want to double check that url will always be a valid param for `URI.create(..)`? I'd think if it isnt, and an IllegalArgumentException is thrown, it would be best just to use the param's original value?

Reply:
Not sure if the externalizer guarantees a valid URL so I just wrapped it in a try/catch